### PR TITLE
[DOC-3075] Add SMP supported features and integrations topic

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add SMP supported features and integrations topic

For reviewers, [a preview](https://646269cefd4e5f0a3c170d84--harness-developer.netlify.app/docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations) is available.

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — COMMENTED** [review]
Made some changes

**Review by pratmit — APPROVED** [review]

File: docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   | Continuous Integration | 2 | 2 | 12 | 0 |
+   | Security Testing Orchestration | 4 | 3| 7 | 0 |
+   | Feature Flags | 3 | 3 | 6 | 0 |
+   | Service Reliability Management | 6 | 8 | 18 | 0 |
+
+### Ingress
+* Istio supported version: 1-15-3
+* NGINX supported version: v1.0.0-alpha.2
+
+### RBAC requirements
```
Comment (StrawberryPad) [inline docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md:27]:
```suggestion
### RBAC requirements

The following permissions are required:
```

File: docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* Permission to deploy Helm
```
Comment (StrawberryPad) [inline docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md:28]:
```suggestion
* Deploy Helm
```

File: docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* Permission to add roles
```
Comment (StrawberryPad) [inline docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md:29]:
```suggestion
* Add roles
```

File: docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* Permission to add/edit secrets
```
Comment (StrawberryPad) [inline docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md:30]:
```suggestion
* Add/edit secrets
```

File: docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md
Comment (StrawberryPad) [inline docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md:1]: General: Looking at the left nav for getting started, it feels like the topics are out of order... like maybe "overviews" should come first.

**Review by StrawberryPad — DISMISSED** [review]

**Pull request merged**